File: app/src/main/java/com/juniperphoton/myerlist/api/response/AddToDoResponse.java
package com.juniperphoton.myerlist.api.response;

import com.google.gson.annotations.SerializedName;
import com.juniperphoton.myerlist.model.ToDo;

public class AddToDoResponse extends CommonResponse {
    @SerializedName("ScheduleInfo")
    private ToDo mToDo;

    public ToDo getToDo() {
        return mToDo;
    }
}

File: app/src/main/java/com/juniperphoton/myerlist/api/response/AddToDoResponse.kt
package com.juniperphoton.myerlist.api.response

import com.google.gson.annotations.SerializedName
import com.juniperphoton.myerlist.model.ToDo

class AddToDoResponse : CommonResponse() {
    @SerializedName("ScheduleInfo")
    val toDo: ToDo? = null
}